sdk/cognitiveservices/service.projects

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+  <ProjectReference Remove="$(MSBuildThisFileDirectory)InkRecognizer.WPF.Stroke\**\*.csproj" />
```

Comment:
ProjectReferences should be indented one more level under the ItemGroup tags.

Reply:
sure, thanks